### PR TITLE
clean up E0718 explanation

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @Dylan-DPC

### CODE REVIEW

File: src/librustc_error_codes/error_codes/E0718.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
-This error indicates that a `#[lang = ".."]` attribute was placed
-on the wrong type of item.
+A `#[lang = ".."]` attribute was placed on the wrong type of item.
```
Comment (pickfire) [inline src/librustc_error_codes/error_codes/E0718.md:1]:
```suggestion
`#[lang = ".."]` attribute was placed on the wrong item type.
```

Comment (GuillaumeGomez) [reply src/librustc_error_codes/error_codes/E0718.md:1]: Why not:

```suggestion
A `#[lang = ".."]` attribute was placed on the wrong item type.
```

?

Comment (pickfire) [reply src/librustc_error_codes/error_codes/E0718.md:1]: `A` feels weird, I don't think we need to quantify that. I mean from my reading point of view, I mean ideally we shouldn't read it aloud when we read (but see instead). Still, I find it shorter, faster and easier to understand without `A` for me.

Comment (Dylan-DPC) [reply src/librustc_error_codes/error_codes/E0718.md:1]: it reads fine to me tbh with `a` rather than without

Comment (GuillaumeGomez) [reply src/librustc_error_codes/error_codes/E0718.md]: I applied the change I suggested then.

**Review by pickfire — APPROVED** [review]
Looks good to me.

**Comment by pickfire** [comment]
@GuillaumeGomez Can I even do rollups? I thought it needs to be authorized users?

**Comment by Dylan-DPC** [comment]
@pickfire this just marks it safe for a rollup, you don't have to do the rollup :D

**Pull request merged**